modules/programs/neomutt.nix

```
@@ -133,9 +133,24 @@ let
     '';
 
   registerAccount = account:
-    with account; ''
+    let
+      mailboxes = if account.neomutt.mailboxName == null then
+        "mailboxes"
+      else
+        ''named-mailboxes "${account.neomutt.mailboxName}"'';
+      extraMailboxes = concatMapStringsSep "\n" (extra:
+        if isString extra then
+          ''mailboxes "${account.maildir.absPath}/${extra}"''
+        else if extra.name == null then
+          ''mailboxes "${account.maildir.absPath}/${extra.mailbox}"''
+        else
+          ''
+            named-mailboxes "${extra.name}" "${account.maildir.absPath}/${extra.mailbox}"'')
```

total nit, not sure if this would even work:
```suggestion
          ''named-mailboxes "${extra.name}" "${account.maildir.absPath}/${extra.mailbox}"'')
```

---

The `format` tool does this